README.md

```diff
@@ -1,2 +1,116 @@
+
+## Azure Pipeline Example
+**Notation sign**: automatically detect the artifact from previous Docker task
+```yaml
+trigger:
+ - master
+pool: 
+  vmImage: 'ubuntu-latest'
+
+steps:
+# login to registry
+- task: Docker@2
+  inputs:
+    containerRegistry: <docker_registry_service_connection>
+    command: 'login'
+# build and push artifact to registry
+- task: Docker@2
+  inputs:
+    repository: <repository_name>
+    command: 'buildAndPush'
+    Dockerfile: './Dockerfile'
+# install notation
+- task: notation@0
+  inputs:
+    command: 'install'
+    version: '1.0.0'
+# automatically detect the artifact pushed by Docker task 
+# and sign the artifact.
+- task: notation@0
+  inputs:
+    version: '1.0.0'
+    command: 'sign'
+    plugin: 'azureKeyVault'
+    azurekvServiceConection: <azurerm_service_connection>
+    keyid: <key_id>
+    selfSigned: true
+```
+
+**Notation verify**
+```yaml
+trigger:
+ - master
+pool: 
+  vmImage: 'ubuntu-latest'
+
+steps:
+# login to registry
+- task: Docker@2
+  inputs:
+    containerRegistry: <docker_registry_service_connection>
+    command: 'login'
+# notation verify
+- task: notation@0
+  inputs:
+    version: '1.0.0'
+    command: 'verify'
+    artifactRefs: '<registry_host>/<repository>@<digest>'
+    trustpolicy: $(Build.SourcesDirectory)/.pipeline/trustpolicy.json
+    truststore: $(Build.SourcesDirectory)/.pipeline/truststore/
+    allowReferrersAPI: true
+```
+
+## Inputs
+`command` - Command  
+`string`. Required. Allowed values: `install`, `sign` and `verify`.
+
+`artifactRefs` - Artifact References  
+`string`. The container artifact reference with digest. If multiple references are used, please use comma to separate them. If it was not specified, the task will automatically detect it from previous Docker task.
+
+`plugin` - Plugin  
+`string`. Required for sign command. Allowed values: `azure-kv`.
+
+`azurekvServiceConnection` - Azure Key Vault Service Connection  
+`string`. Required for `azure-kv` plugin. The Azure Resource Manager service connection for accessing Azure Key Vualt.
+
+`keyid` - Key ID  
+`string`. Required for `azure-kv` plugin. The key identifier of an Azure Key Vault certificate.
+
+`selfSigned` - Self signed  
+`boolean`. Whether the certficate is self-signed certificate.
+
+`cacerts` - Certificate Bundle File Path  
+`string`. The certificate bundle file containing intermidiate certificates and root certificate.
+
+`trustpolicy` - Trust Policy File Path  
+`string`. Required for `verify` command. The trust policy file path.
+
+`truststore` - Trust Store Folder Path  
+`string`. Requried for `verify` command. The trust store folder path.
+
+`signatureFormat` - Signature Format  
+`string`. Signature envelope format. Allowed values: `jws`, `cose`.
+
+`allowReferrersAPI` - [Experimental] Allow Referrers API  
+`boolean`. Use the Referrers API to sign signatures, if not supported (returns 404), fallback to the Referrers tag schema.
```

Comment:
I've noticed that the casing of variable names are inconsistent. For examples,
- `trustpolicy` v.s.`trustPolicy`
- `selfsigned` v.s. `selfSigned`
- ...

/cc @FeynmanZhou @yizha1

Reply:
Updated to be `trustPolicy`, `trustStore` and `selfSigned` to follow the ADO task convention